### PR TITLE
Remove duplicate `"font": "regular"` fields from translation files

### PR DESCRIPTION
**Pull request by GnomedDev**

### CODE REVIEW

File: tools/bmg-cleaner/bmg-cleaner.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import json, json5
```
Comment (stblr) [inline tools/bmg-cleaner/bmg-cleaner.py:1]: The first import is unused.

File: tools/bmg-cleaner/bmg-cleaner.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    assert isinstance(bmg, dict)    
+
+    for message_id, message in bmg.items():
+        font = message.get("font")
+
+        new_bmg[message_id] = {"string": message["string"]}
+        if font is not None and font != "regular": 
+            new_bmg[message_id]["font"] = font
+
+    path.write_text(json5.dumps(new_bmg, indent=4, ensure_ascii=False))
```
Comment (stblr) [inline tools/bmg-cleaner/bmg-cleaner.py:22]: What about `quote_keys`?

**Review by stblr — CHANGES_REQUESTED** [review]

File: tools/bmg-cleaner/bmg-cleaner.py (hunk 2 above)
Comment (GnomedDev) [reply tools/bmg-cleaner/bmg-cleaner.py:22]: What about it? If we are using json5, shouldn't we be taking advantage of it's features?

Comment (stblr) [reply tools/bmg-cleaner/bmg-cleaner.py:22]: Yeah why not.

**Review by stblr — APPROVED** [review]

**Pull request merged**